### PR TITLE
docs: add screenshot and install link

### PR DESCRIPTION
**Pull request by rothgar**
Simplifies begining paragraphs and fixes some spelling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,22 +16,24 @@
...
 
-You provide the machines – edge compute, bare metal, VMs, or in your cloud account. Boot from an Omni image. Click to allocate to a cluster. That’s it!
+Boot from an Omni image. Click to allocate to a cluster. That’s it!
 
 - Vanilla Kubernetes, on your machines, under your control.
 - Elegant UI for management and operations
 - Security taken care of – ties into your Enterprise ID provider
 - Highly Available Kubernetes API endpoint built in
-- Firewall friendly. Manage Edge nodes securely
+- Firewall friendly—manage edge nodes securely
```
Comment (Unix4ever) [inline README.md:28]:
```suggestion
- Firewall friendly — manage edge nodes securely
```

**Review by Unix4ever — APPROVED** [review]
Especially like `lolfart` account name on the screenshot of Omni :D

File: README.md (hunk 1 above)
Comment (rothgar) [reply README.md:28]: spaces around em dashes is not recommended in most style guides

Comment (Unix4ever) [reply README.md:28]: then we might probably remove them on line 26 ;)

**Comment by rothgar** [comment]
moved to #96

**Pull request closed without merging**